typescript/src/market-proxy/parser/getOrderRequest.ts

```diff
@@ -54,5 +55,10 @@ export const getOrderRequest = ({
     log('order is missing symbol or tid or legs');
   }
 
+  if (expireTimestamp !== undefined)
```

Comment:
sorry the `{` needs to be on the same line as `if` or it can create issue in certain scenarios in JS

```
Styling with Markdown is supported
Start a review
Add single comment
Cancel
  if (expireTimestamp !== undefined) {
    payload.expire_timestamp = expireTimestamp.toString()
  }
```